system-addon/content-src/components/TopSites/TopSites.jsx

```diff
@@ -120,6 +124,9 @@ export class _TopSites extends React.PureComponent {
               <div className="modal">
                 <TopSiteForm
                   site={props.TopSites.rows[editForm.index]}
+                  screenshotPreview={this.props.TopSites.editForm.screenshotPreview}
+                  screenshotRequestFailed={this.props.TopSites.editForm.screenshotRequestFailed}
+                  customScreenshotURL={this.props.TopSites.editForm.customScreenshotURL}
```

Comment:
Maybe this seems to point to just directly connecting TopSiteForm to the store instead of passing it through this component?